gremlin-go/driver/bytecode.go

```diff
@@ -122,16 +126,24 @@ func (bytecode *bytecode) convertArgument(arg interface{}) (interface{}, error)
 		return newSlice, nil
 	default:
 		switch v := arg.(type) {
-		case binding:
-			convertedValue, err := bytecode.convertArgument(v.value)
+		case *Binding:
+			convertedValue, err := bytecode.convertArgument(v.Value)
 			if err != nil {
 				return nil, err
 			}
-			bytecode.bindings[v.key] = v.value
-			return binding{
-				key:   v.key,
-				value: convertedValue,
+			bytecode.bindings[v.Key] = v.Value
+			return &Binding{
+				Key:   v.Key,
+				Value: convertedValue,
 			}, nil
+		case *GraphTraversal:
```

Comment:
Can we combine this and the Traversal case?

Reply:
Actually yes, I'll need to change some test cases in `bytecode`, but they can both be under `*Traversal`

Reply:
I'm thinking of removing the `Traversal` case entirely, as we actually don't return `Traversal`/`*Traversal` at any point, just `*GraphTraversals`.